javascript/index.js

```diff
@@ -48,18 +80,63 @@ function setSplitBtn() {
 
 function setStartBtn() {
   // ... your code goes here
+  
 }
 
 function setResetBtn() {
   // ... your code goes here
 }
 
 // Start/Stop Button
-btnLeft.addEventListener('click', () => {
-  // ... your code goes here
-});
+
+ document.getElementById('btnLeft').addEventListener("click", () => {
+  const btnLeftClassName = document.getElementById('btnLeft').className;
+  if(btnLeftClassName==="btn start"){
+   chronometer.startClick(printTime);
+  btnLeft.innerHTML="STOP";
+  btnLeft.className="btn stop"
+  btnRight.className="btn split"
+  btnRight.innerHTML="SPLIT"
+  chronometer.stopClick()
+  }
+
+ if(btnLeftClassName==="btn stop"){
+  chronometer.stopClick()
+  btnLeft.innerHTML="STOP";
+  btnLeft.className="btn stop"
+  btnRight.className="btn reset"
+  btnRight.innerHTML="RESET"
+  chronometer.resetClick();
+ }
+  });
+
+
+
+
 
 // Reset/Split Button
 btnRight.addEventListener('click', () => {
   // ... your code goes here
+  const btnRighttClassName = document.getElementById('btnRight').className;
+  if(btnRighttClassName==="btn reset"){
+  btnRight.innerHTML="SPLIT";
+  btnLeft.className="btn stop";
+  btnRight.className="btn split";
+  btnLeft.innerHTML="STOP"
+   chronometer.resetClick();
+    clearSplits();
+  
+   }
+
+  if(btnRighttClassName==="btn split"){
+    printSplit();
+    btnRight.innerHTML="RESET";
```

Comment:
After you click on "SPLIT", the only thing that should happen is executing `printSplit()` so the current time gets added to the list.

You don't need line 133-136, that behaviour is taken care of already with the btnLeft event listener.